index.html

```diff
@@ -0,0 +1,449 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <!--Mobile menu-->
+      <div class="menu-bars">
+        <i class="fas fa-bars open-btn"></i>
```

Comment:
- [ ] The menu hamburger looks good. nice job. lets position it well so it looks like the one on the original design.
your design 👇🏾
![Screenshot (9)](https://user-images.githubusercontent.com/65298048/169404603-231b6951-850a-49ba-989e-57fd0d4e3497.png)

--------
Original [figma 
 ](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) design 👇🏾 
-------
![Screenshot (8)](https://user-images.githubusercontent.com/65298048/169404573-828e5e3a-5df5-4eeb-a961-0fc488eb1797.png)